packages/manager/apps/pci-private-registry/src/pages/delete/Delete.page.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import { act, fireEvent, render } from '@testing-library/react';
+import { describe, it, vi } from 'vitest';
+import { UseMutationResult, UseQueryResult } from '@tanstack/react-query';
+import {
+  OdsInputValueChangeEventDetail,
+  OsdsInput,
+} from '@ovhcloud/ods-components';
+import DeletePage from '@/pages/delete/Delete.page';
+import { wrapper } from '@/wrapperRenders';
+import * as ApiRegistryModule from '@/api/hooks/useRegistry';
+import { TRegistry } from '@/api/data/registry';
+
+vi.mock('@ovhcloud/manager-components', () => ({
+  useNotifications: vi.fn().mockReturnValue({
+    addError: vi.fn(),
+    addSuccess: vi.fn(),
+  }),
+}));
```

Comment:
It's not already mocked globally on setupTests file ?

Reply:
now yes :)